linkis-engineconn-plugins/spark/pom.xml

```diff
@@ -187,12 +187,21 @@
       <artifactId>linkis-rpc</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>net.sf.py4j</groupId>
+      <artifactId>py4j</artifactId>
```

Comment:
Here py4j is compatible and should not be introduced separately, because spark jars already have this dependency, which will cause low version conflicts

Reply:
have set the dependency as provided